ui/src/components/pipeline/Pipeline.tsx

```diff
@@ -98,7 +99,7 @@ export function Pipeline() {
       Promise.allSettled(
         pipeline?.spec?.vertices.map((vertex) => {
           return fetch(
-            `/api/v1/namespaces/${namespaceId}/pipelines/${pipelineId}/vertices/${vertex.name}/pods`
+            getBaseHref() + `/api/v1/namespaces/${namespaceId}/pipelines/${pipelineId}/vertices/${vertex.name}/pods`
```

Comment:
Does using `./api/v1/xxxx` work?

Reply:
This approach doesn't work currently, this is what the request path becomes:
```
https://localhost:8443/namespaces/numaflow-system/pipelines/api/v1/namespaces/numaflow-system/pipelines/simple-pipeline?refreshKey=1681937373756
```